addons/hw_drivers/tools/helpers.py

```diff
@@ -92,6 +92,7 @@ def check_certificate():
     server = get_odoo_server_url()
 
     if not server:
+        _logger.info('The IoT Box does not have an associated Odoo database')
```

Comment:
(same as before) generic logs have generally no point. I would say that this log is useless with the one you modified above. It just add information on something we can deduce